src/treespec/treespec.cpp

```diff
@@ -288,8 +382,8 @@ py::list PyTreeSpec::Entries() const {
         return py::list{root.node_entries};
     }
     switch (root.kind) {
-        case PyTreeKind::None:
-        case PyTreeKind::Leaf: {
+        case PyTreeKind::Leaf:
```

Comment:
Why change this?

Reply:
I reorder the switch cases to match the order of the enum definitions.